src/data/finset/basic.lean

```diff
@@ -2065,6 +2073,9 @@ by simp_rw [to_finset, dedup_idempotent]
   (m.dedup.bind f).to_finset = (m.bind f).to_finset :=
 by simp_rw [to_finset, dedup_bind_dedup]
 
+instance is_well_founded_ssubset : is_well_founded (multiset α) (⊂) :=
```

Comment:
Please fix the lint error, and this should be `finset`.

Reply:
What do you mean? The `finset` instance is above.

Reply:
Oh I see, what you have is correct (and you need the proof because ssubset isn't the same as lt on multiset). Sorry for the noise.